scripts/spring-dash/grafana-datasource.yml

```diff
@@ -4,4 +4,4 @@ datasources:
 - name: prometheus
   type: prometheus
   access: direct
-  url: http://10.200.10.1:9090
+  url: https://10.200.10.1:9090
```

Comment:
HTTPS here certainly won't work since this is really pointing at localhost. I tried it with `http://localhost:9090` and it worked fine for me. I'm not sure why the alias IP is needed. @jkschneider

Reply:
Maybe this is a difference of using a docker-machine VM vs native OS support. Now that MacOS has native support, I'm not sure we need this setup.

Reply:
> Now that MacOS has native support
If that's true, then I think you're right!